lib/internal/streams/async_iterator.js

```diff
@@ -20,6 +20,16 @@ const kLastPromise = Symbol('lastPromise');
 const kHandlePromise = Symbol('handlePromise');
 const kStream = Symbol('stream');
 
+let Readable;
+
+function destroy(stream, err) {
+  // request.destroy just do .end - .abort is what we want
+  if (typeof stream.abort === 'function') return stream.abort();
+  if (stream.req && stream.req.abort === 'function') return stream.req.abort();
```

Comment:
Sorry, you forget `typeof` here after `&&`.

Reply:
Thank you!